src/gtest/wrappers.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * wrappers.h - Function Wrapper Declarations for Google Mock Unit Tests
+ *
+ * PURPOSE:
+ * This file declares C function wrappers that enable mocking of Valkey C functions
+ * in C++ Google Mock unit tests. It bridges C code with C++ test infrastructure.
```

Comment:
Change `C++ Google Mock` to `GoogleTest`.
Also, "It bridges C code with the gtest infrastructure"